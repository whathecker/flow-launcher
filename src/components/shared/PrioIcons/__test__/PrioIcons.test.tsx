import * as React from "react";
import renderer from "react-test-renderer";
import {
  EmptyPrioIcon,
  HighestPrioIcon,
  HighPrioIcon,
  MidPrioIcon,
  LowPrioIcon,
} from "../PrioIcons";

describe("Test PrioIcons component", () => {
  it("Should renders all icons correctly", () => {
    const tree1 = renderer
      .create(<EmptyPrioIcon style={{ width: 32, height: 32 }} />)
      .toJSON();

    const tree2 = renderer
      .create(<HighestPrioIcon style={{ width: 32, height: 32 }} />)
      .toJSON();

    const tree3 = renderer
      .create(<HighPrioIcon style={{ width: 32, height: 32 }} />)
      .toJSON();

    const tree4 = renderer
      .create(<MidPrioIcon style={{ width: 32, height: 32 }} />)
      .toJSON();

    const tree5 = renderer
      .create(<LowPrioIcon style={{ width: 32, height: 32 }} />)
      .toJSON();

    expect(tree1).toMatchSnapshot();
    expect(tree2).toMatchSnapshot();
    expect(tree3).toMatchSnapshot();
    expect(tree4).toMatchSnapshot();
    expect(tree5).toMatchSnapshot();
  });
});
