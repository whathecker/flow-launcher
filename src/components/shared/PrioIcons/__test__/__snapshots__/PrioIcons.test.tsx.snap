// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PrioIcons component Should renders all icons correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 32,
      "width": 32,
    }
  }
/>
`;

exports[`Test PrioIcons component Should renders all icons correctly 2`] = `
<Image
  source={1}
  style={
    Object {
      "height": 32,
      "width": 32,
    }
  }
/>
`;

exports[`Test PrioIcons component Should renders all icons correctly 3`] = `
<Image
  source={1}
  style={
    Object {
      "height": 32,
      "width": 32,
    }
  }
/>
`;

exports[`Test PrioIcons component Should renders all icons correctly 4`] = `
<Image
  source={1}
  style={
    Object {
      "height": 32,
      "width": 32,
    }
  }
/>
`;

exports[`Test PrioIcons component Should renders all icons correctly 5`] = `
<Image
  source={1}
  style={
    Object {
      "height": 32,
      "width": 32,
    }
  }
/>
`;
